packages/peregrine/src/index.js

```diff
@@ -5,6 +5,6 @@ export { default as ContainerChild } from './ContainerChild';
 export { default as List, Items, Item } from './List';
 export { default as Page } from './Page';
 export { default as Price } from './Price';
-export { default as Router } from './Router';
+export { default as Router, RouteConsumer } from './Router';
```

Comment:
This is a solid idea, but it shouldn't be necessary since we upgraded React Router in #639.

Reply:
shall we use withRouter API